multivac/gather_data.py

```diff
@@ -130,14 +130,36 @@ def get_test_data(log_file) -> list:
                     test_attempt = test['test_attempt'] + 1
                     break
 
+            # Detect type of the test
+            test_type_name = test_name.split('/')[0]
+            subtype = 'None'
+            if test_type_name.endswith('tap'):
+                test_type = 'tap'
+            elif test_type_name.endswith('luatest'):
+                test_type = 'luatest'
+            else:
+                test_type = 'diff'
+                if test_type.endswith('py'):
+                    subtype = 'python'
+                elif 'sql' in test_type:
+                    subtype = 'sql'
             test_record = {'name': test_name,
                            'conf': conf or 'none',
+                           'test_type': test_type,
+                           'subtype': subtype,
                            'test_attempt': test_attempt}
             test_attempt = 1
             tests_data.append(test_record)
 
         return tests_data
 
+    @staticmethod
+    def get_release_or_debug(log_file):
+        for line in log_file:
+            if 'Run make -f' in line and line.endswith('debug\n'):
```

Comment:
I am not sure that is the right way to get `release` or `debug` build. You rely on make target name here, which is fragile. One day we can rename these targets and your logic will be broken.

Reply:
Fixed